doc/rados/configuration/osd-config-ref.rst

```diff
@@ -654,7 +654,7 @@ perform well in a degraded state.
               increased load on the cluster.
 
 :Type: 32-bit Integer
-:Default: ``15``
+:Default: ``3``
```

Comment:
Hi Kefu, 

I reverted that merge commit and fixed the commit message. However, I see that there are even more problems now. I think I'll just fork the repo again and create a new pull request..